kaptan/__init__.py

```diff
@@ -52,20 +53,39 @@ def upsert(self, key, value):
         self.configuration_data.update({key: value})
         return self
 
+    def _is_python_file(self, value):
```

Comment:
Can we get some documentation as to what "value" is supposed to look like? i.e. is it a full file path as a string (``/path/to/file.ext``), or is it just the file (``file.ext``).